convert-falcon-hf-to-gguf.py

```diff
@@ -161,24 +161,9 @@ def parse_args() -> argparse.Namespace:
 byte_decoder = {v: k for k, v in byte_encoder.items()}
 
 for i in range(vocab_size):
-    if i in reverse_vocab:
-        try:
-            text = bytearray([byte_decoder[c] for c in reverse_vocab[i]])
-        except KeyError:
```

Comment:
MPT seems to need this KeyError handler. Is this problem solved a different way here, or simply ignored?
see https://github.com/ggerganov/llama.cpp/pull/3021#issuecomment-1738041891

Reply:
It is solved in a different way: the vocabulary is moved unmodified to `llama.cpp` and byte encoding and decoding are done in C++.